yelp_package/dockerfiles/itest/marathon/Dockerfile

```diff
@@ -29,12 +29,8 @@ RUN echo "debconf shared/accepted-oracle-license-v1-1 select true" | debconf-set
 RUN echo "debconf shared/accepted-oracle-license-v1-1 seen true" | debconf-set-selections
 RUN apt-get update && apt-get -y install lsb-release oracle-java8-installer
 
-RUN apt-get -y install unzip
-RUN wget https://s3.amazonaws.com/repos.mesosphere.com/ubuntu/pool/main/m/marathon/marathon_1.4.0-1.0.562.ubuntu1404_amd64.deb \
-    && dpkg -i marathon_1.4.0-1.0.562.ubuntu1404_amd64.deb && rm marathon_1.4.0-1.0.562.ubuntu1404_amd64.deb
+RUN apt-get update && apt-get -y --allow-unauthenticated install marathon=1.4.0-1.0.631.ubuntu1404
```

Comment:
We already add the key at https://github.com/Yelp/paasta/blob/master/yelp_package/dockerfiles/itest/marathon/Dockerfile#L21 . I've gone ahead and removed the `--alow-unauthenticated`.